src/virt-install

```diff
@@ -197,6 +207,8 @@ http_proc = None
 have_virtinstall_2 = int(subprocess.check_output(['virt-install', '--version'],
                                                  stderr=subprocess.STDOUT).strip().decode('UTF-8').split('.')[0]) >= 2
 
+ks_tmp.flush()
```

Comment:
I moved this way down here since I lost 20 minutes debugging why the above fragment wasn't working, since I had the flush before it.